### PR TITLE
updated the ISBN description for additonal clarity

### PR DESCRIPTION
**Pull request by blakeashleyjr**
This PR is based on the discussion of [this issue.](https://github.com/exercism/problem-specifications/issues/1975)

When using the default code blocks with the text language, the subscript characters are removed. I switched them to block quotes to preserve the subscripts and everything looks good because it is only text anyways. Let me know if this violates a standard somewhere

### CODE REVIEW

**Comment by blakeashleyjr** [comment]
The linter failed because it is confusing the "*"s with emphasis marks when they represent multiplication symbols.

File: exercises/isbn-verifier/description.md [hunk 1]
```diff
@@ -5,21 +5,21 @@ numbers. These normally contain dashes and look like: `3-598-21508-8`
 
 ## ISBN
 
-The ISBN-10 format is 9 digits (0 to 9) plus one check character (either a digit or an X only). In the case the check character is an X, this represents the value '10'. These may be communicated with or without hyphens, and can be checked for their validity by the following formula:
+The ISBN-10 format is 9 digits (0 to 9, represented below with a subscript) plus one check character (either a digit or an X only). In the case the check character is an X, this represents the value '10'. These may be communicated with or without hyphens, and can be checked for their validity by the following formula:
+
+
+> (1₁ * 10 + 2₂ * 9 + 3₃ * 8 + 4₄ * 7 + 5₅ * 6 + 6₆ * 5 + 7₇ * 4 + 8₈ * 3 + 9₉ * 2 + 10₁₀ * 1) mod 11 == 0
```
Comment (SaschaMann) [inline exercises/isbn-verifier/description.md:11]: This notation is really confusing. I only know digits with indices like this where the index clarifies which base the number is in. I think it was suggested that these should be `d` with subscripts?

**Review by SaschaMann — CHANGES_REQUESTED** [review]

**Comment by SaschaMann** [comment]
> The linter failed because it is confusing the "*"s with emphasis marks when they represent multiplication symbols.

You can escape them with a backslash. But I don't understand why the text block removes subscripts. Did you use the Unicode subscripts or `<sub>`?

**Comment by blakeashleyjr** [comment]
Thanks @SaschaMann, I corrected the markdown according to your suggestion and switched the digits to a "d" for more clarity. Let me know what you think

The text block removed the Unicode subscripts and <sub> tags for me in VSCode. Interestingly, the tags do work outside of a text block though:

image

I can switch to using <sub> tags but they are likely less supported than the Unicode characters.

**Comment by angelikatyborska** [comment]
> The text block removed the Unicode subscripts [...] for me in VSCode

Are you sure? They do work for me (a fresh VSC installation on a m1 mac). Here's screenshots of VSC and RubyMine rendering the subscript unicode characters in a text code block:
Screenshot 2022-03-14 at 17 16 32
Screenshot 2022-03-14 at 17 16 53

File: exercises/isbn-verifier/description.md [hunk 2]
```diff
@@ -7,19 +7,15 @@ numbers. These normally contain dashes and look like: `3-598-21508-8`
 
 The ISBN-10 format is 9 digits (0 to 9) plus one check character (either a digit or an X only). In the case the check character is an X, this represents the value '10'. These may be communicated with or without hyphens, and can be checked for their validity by the following formula:
 
-```text
-(x1 * 10 + x2 * 9 + x3 * 8 + x4 * 7 + x5 * 6 + x6 * 5 + x7 * 4 + x8 * 3 + x9 * 2 + x10 * 1) mod 11 == 0
-```
+> (d₁ \* 10 + d₂ \* 9 + d₃ \* 8 + d₄ \* 7 + d₅ \* 6 + d₆ \* 5 + d₇ \* 4 + d₈ \* 3 + d₉ \* 2 + d₁₀ \* 1) mod 11 == 0
```
Comment (SaschaMann) [inline exercises/isbn-verifier/description.md:10]:
```suggestion
```text
(d₁ * 10 + d₂ * 9 + d₃ * 8 + d₄ * 7 + d₅ * 6 + d₆ * 5 + d₇ * 4 + d₈ * 3 + d₉ * 2 + d₁₀ * 1) mod 11 == 0
```
```

File: exercises/isbn-verifier/description.md [hunk 3]
```diff
@@ -7,19 +7,15 @@ numbers. These normally contain dashes and look like: `3-598-21508-8`
...
 If the result is 0, then it is a valid ISBN-10, otherwise it is invalid.
 
 ## Example
 
 Let's take the ISBN-10 `3-598-21508-8`. We plug it in to the formula, and get:
 
-```text
-(3 * 10 + 5 * 9 + 9 * 8 + 8 * 7 + 2 * 6 + 1 * 5 + 5 * 4 + 0 * 3 + 8 * 2 + 8 * 1) mod 11 == 0
-```
+> (3 \* 10 + 5 \* 9 + 9 \* 8 + 8 \* 7 + 2 \* 6 + 1 \* 5 + 5 \* 4 + 0 \* 3 + 8 \* 2 + 8 \* 1) mod 11 == 0
```
Comment (SaschaMann) [inline exercises/isbn-verifier/description.md:18]:
```suggestion
```text
(3 * 10 + 5 * 9 + 9 * 8 + 8 * 7 + 2 * 6 + 1 * 5 + 5 * 4 + 0 * 3 + 8 * 2 + 8 * 1) mod 11 == 0
```
```

**Review by SaschaMann — COMMENTED** [review]
> Are you sure? They do work for me (a fresh VSC installation on a m1 mac). Here's screenshots of VSC and RubyMine rendering the subscript unicode characters in a text code block:

Yeah they work fine for me too.

When we get the equation support that was promised as a launch feature, this would be a great candidate to convert to that.

**Comment by blakeashleyjr** [comment]
Thank you @SaschaMann and @angelikatyborska for testing that. I do have a couple markdown specific plugins enabled that may be interfering with the text box specifically.

I changed the description according to your suggestions. Let me know what you think now.

**Review by SaschaMann — APPROVED** [review]
Thanks!

@kotp talking about the website/v3 launch.

**Review by angelikatyborska — APPROVED** [review]

**Review by ErikSchierboom — APPROVED** [review]

**Pull request merged**